Core/src/org/sleuthkit/autopsy/logicalimager/configuration/ConfigureLogicalImager.java

```diff
@@ -41,18 +43,20 @@
         category = "Tools",
         id = "org.sleuthkit.autopsy.configurelogicalimager.ConfigureLogicalImager"
 )
-@ActionRegistration(
-        displayName = "#CTL_ConfigureLogicalImager"
-)
+@ActionRegistration(displayName = "#CTL_ConfigureLogicalImager", lazy = false)
 @ActionReference(path = "Menu/Tools", position = 2000, separatorBefore = 1999)
 @Messages("CTL_ConfigureLogicalImager=Configure Logical Imager")
-public final class ConfigureLogicalImager implements ActionListener {
+public final class ConfigureLogicalImager extends CallableSystemAction {
+
+    private static final String DISPLAY_NAME = Bundle.CTL_ConfigureLogicalImager();
```

Comment:
I think Will has changed the display name for this action so there should not be anything to do, but please coordinate with him.